UnitsNet/UnitDefinitions/FoulingFactor.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Name": "FoulingFactor",
+  "BaseUnit": "HourSquareFeetDegreefahrenheitPerBtu",
+  "XmlDoc": "Heat Transfer Coefficient or Thermal conductivity - indicates a materials ability to conduct heat.",
+  "Units": [
+    {
+      "SingularName": "HourSquareFeetDegreefahrenheitPerBtu",
+      "PluralName": "HourSquareFeetDegreefahrenheitsPerBtu",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "hr*ft2�F/Btu" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SquareMeterDegreeCentigradePerWatt",
```

Comment:
That is fine. And also HourSquareFeetDegreeFahrenheitPerBtu can be change to 
HourSquareFeetFahrenheitPerBtu as Celsius and Fahrenheit are related to temperature.